### ISSUE TITLE
Blocks saved in Customizer > Widgets, but not showing up in Appearance > Widgets.

### ISSUE THREAD

**Issue by danielk-am** [Automattic/wp-calypso#59852]
### Quick summary

https://user-images.githubusercontent.com/92185881/148543238-7dde144d-c00c-4d42-afa2-d11e9b16d55c.mp4

A screen recording of the issue above (2m 33s).

### Steps to reproduce

1. Customizer > Widgets
2. Add image block. Save.
3. Appearance > Widgets
4. Blocks saved in Customizer were not appearing. Added new blocks, the Editor froze.

### What you expected to happen

Blocks saved on either end to show up on both Customizer > Widgets and Appearance > Widgets.

### What actually happened

Blocks saved in Customizer > Widgets, but not showing up in Appearance > Widgets.

Tried adding new Blocks in Appearance > Widgets, the Editor froze.

### Context

4557903-zen -- The user first encountered this issue. Initially linked to another earlier bug report that got fixed (https://github.com/Automattic/wp-calypso/issues/56922). 

4665129-zen -- The user returned yesterday and informed us that the issue wasn't resolved.

As I was not able to replicate the issue on my side, and noticed that the user invited us to try adding the Blocks on their site in the previous Ticket, I then tested it on their end and confirmed that the issue still persists.

### Simple, Atomic or both?

Simple

**Comment by sashastone** [comment]
I was curious if this was an issue with the theme. I tested it out with Twenty Fifteen, and had a similar issue: https://d.pr/v/ptTA2d

**Comment by the-misha** [comment]
This may be related: 4690142-zd-woothemes

User had an un-deletable legacy Custom HTML widget showing in a footer widget area in _Appearance → Widgets_ which didn't display in the Customizer or on the live site. They edited the widget areas in the Customizer, which removed the duplicate.

**Comment by kimerlin81** [comment]
Thanks for submitting this bug report @1dr0!

I was unable to replicate this on my test site which was set to the same theme, plan and customizer settings as the user.

While this issue does seem to be specific to just this user, I don't believe it's theme-specific. I tested by changing this users theme to several other themes including 2015 and Lovecraft and I found the behavior to be the same:

https://user-images.githubusercontent.com/83732167/150606089-26372b89-a360-4d5f-a28c-a44c6d92605c.mp4

Since we're unable to replicate this issue and it appears to only be affecting this user, I am going to close this bug report and place any additional findings in the P2 that Sasha created.
